Editor/LyumaAv3EditorSupport.cs

```diff
@@ -321,30 +284,47 @@ static LyumaAv3EditorSupport()
 					{
 						_preprocessAvatarCallbacks = callbacks;
 						lockMaterials = _preprocessAvatarCallbacks.FirstOrDefault(x => x.GetType().Name == "LockMaterialsOnUpload");
+						vrcFuryPreprocess = _preprocessAvatarCallbacks.FirstOrDefault(x => x.GetType().FullName == "VF.VrcHooks.PreuploadHook");
 					}
 				}
 
 				if (lockMaterials != null)
 				{
 					_preprocessAvatarCallbacks.Remove(lockMaterials);
 				}
+
```

Comment:
AH. I understand what you were mentioning before. I was under the impression that VRCF would not fail here because at this point in the process, there would be no vrcf components remaining on the avatar. It is a bug that they are remaining. I'll fix that, so this special case should not be needed.

Reply:
VRCF doesn't fail, it displays a warning when the preprocess hook is thrown, no matter if any VRCF components  exist at all

Reply:
Yep, that was a mistake and has been fixed in https://github.com/VRCFury/VRCFury/commit/a4e8039fd3df5e05a05c2217cbd1e0b71dafdb57